Library/Formula/websockify.rb

```diff
@@ -0,0 +1,70 @@
+require "FileUtils"
+
+class Websockify < Formula
+  homepage "https://github.com/kanaka/websockify"
+  url "https://github.com/kanaka/websockify/archive/v0.6.0.tar.gz"
+  sha1 "f2bffebc9e4d94ad3a329638bee63e899eaad9c0"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
```

Comment:
Check the upstream Git for either a ` requirements.txt ` file or if that doesn't exist open the ` setup.py ` file and see if any other Python modules are listed as required. I'd be surprised if there was nothing as a resource needed here.